tasks/go.py

```diff
@@ -23,7 +23,6 @@
 }
 GOARCH_MAPPING = {
     "x64": "amd64",
-    "x86": "386",
     "arm64": "arm64",
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution. -->
**❓ question**
We never use `arm64` arch in tasks?

Reply:
We do use it [here](https://github.com/DataDog/datadog-agent/blob/b0d911fcf77f9e1e4bfa2045137d42a40361a5be/tasks/diff.py/#L104) !